src/Whoops/Run.php

```diff
@@ -74,7 +74,7 @@ public function __construct(SystemFacade $system = null)
     /**
      * Explicitly request your handler runs as the last of all currently registered handlers.
      *
-     * @param HandlerInterface $handler
+     * @param Callable|HandlerInterface $handler
```

Comment:
you should also adjust the called `resolveHandler` methods phpdoc

Reply:
Thank you!
I fixed it at [`59f635b` (#701)](https://github.com/filp/whoops/pull/701/commits/59f635b59788cab37a24c77b9a723d0cc9cc86a1)

Reply:
@fujiwaraizuho Since [`callable`](https://www.php.net/manual/en/language.types.callable.php) is a type, not a class, it is usually written in all lowercase.

Reply:
fixed! [`198e712` (#701)](https://github.com/filp/whoops/pull/701/commits/198e7120b4bb26204010ae80a11f9b77c641359d)